tests/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+if(EMSCRIPTEN)
+    message(STATUS "Tests don't work with emscripten -> skipping tests")
```

Comment:
WIP: Refactored this, so that loadtests get built for web. I still get a runtime error (cannot create GL context) though.
Is there a list of formats to exclude from emscripten? If not, I'll look it up in the gyp config.